### PR TITLE
Updated - and greatly simplified - GenerateAssertionsResponse. Now ju…

### LINKED ISSUE #173: Change generate-assertions http response to status only
See meeting notes - assume the dashboard gets info from the database and instead just give it a 4-valued ENUM to tell it whether assertion generation succeeded or failed, and whether it's worth redoing.

### PR DESCRIPTION
**Pull request by vteague**
…st the contest name and two booleans: did it succeed? and is it worth retrying?

This makes everything a lot simpler, though it will get more complicated again when we want to retrieve winners and errors from the database.

### CODE REVIEW

**Comment by vteague** [comment]
This matches the corresponding PR in the raire-service.

**Review by cjcarlton — APPROVED** [review]
Changes look good and the endpoint worked when i tested it. thanks!

File: server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertions.java [hunk 1]
```diff
@@ -177,22 +175,20 @@ public String endpointBody(final Request the_request, final Response the_respons
    * - Gather all the IRVContestResults
    * - For each IRV contest, make a request to the raire-service get-assertions endpoint of the right format type
    * - Collate all the results into a list.
+   *
    * @param IRVContestResults the collection of all IRV ContestResults.
    * @param timeLimitSeconds  the time limit for raire assertion generation, per contest.
    * @param raireUrl          the url where the raire-service is running.
```
Comment (michelleblom) [inline server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertions.java:181]: I'm guessing these input arguments can be final?

File: server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertions.java [hunk 2]
```diff
@@ -202,21 +198,23 @@ protected List<GenerateAssertionsResponseWithErrors> generateAllAssertions(List<
...
    * @param IRVContestResults The list of all ContestResults for IRV contests. Note that these do
    *                          not have the correct winners or losers for IRV.
    * @param contestName       The name of the contest.
    * @param timeLimitSeconds  The time limit allowed for raire to compute the assertions (not
    *                          counting time taken to retrieve vote data from the database).
    * @param raireUrl          The url of the raire service.
-   * @return                  The GenerateAssertionsResponseWithErrors, which usually contains a
-   *                          winner but may instead be UNKNOWN_WINNER and an error message.
+   * @return The GenerateAssertionsResponseWithErrors, which usually contains a
+   * winner but may instead be UNKNOWN_WINNER and an error message.
```
Comment (michelleblom) [inline server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertions.java:214]: Check input arguments for what can be final.

**Review by michelleblom — APPROVED** [review]

**Pull request merged**